onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -4046,25 +4046,17 @@ def aten_index_put(
 ) -> TReal:
     """index_put(Tensor self, Tensor?[] indices, Tensor values, bool accumulate=False) -> Tensor"""
 
-    index = op.SequenceAt(indices, 0)  # assume indices only have 1 element
-    # change array([1,3]) to array([[1,1,1,1,1],[3,3,3,3,3]])
-    self_dim_1 = op.Gather(op.Shape(self), 1)
-    index_dim_0 = op.Gather(op.Shape(index), 0)
-    neg_1 = op.Constant(value_ints=[-1])
-    shape = op.Concat(op.Reshape(self_dim_1, neg_1), op.Reshape(index_dim_0, neg_1), axis=0)
-    new_ind = op.Expand(index, shape)
-    new_ind_t = op.Transpose(new_ind)
+    index = op.SequenceAt(indices, 0)
+    new_index = op.Unsqueeze(index, -1)
```

Comment:
nit: I wonder if `Unsqueeze(index, [-1])` is more precise. My understanding is that the second argument should be a 1-dimensional tensor, but `-1` will produce a scalar -1. Even if it works with ORT, it may be better to stick closer to expectation.

Reply:
done